packages/nuxt/src/module.ts

```diff
@@ -27,7 +27,7 @@ const module: NuxtModule<ModuleOptions> = defineNuxtModule<ModuleOptions>({
     name: 'pinia',
     configKey: 'pinia',
     compatibility: {
-      nuxt: '^3.15.0',
+      nuxt: '>=3.15.0',
```

Comment:
Let's keep it a bit stricter:

```suggestion
      nuxt: '^3.15.0 || ^4.0.0-0',
```

Reply:
You’re right, thank you very much!